BetonQuest-core/src/main/java/pl/betoncraft/betonquest/config/ConfigPackage.java

```diff
@@ -17,17 +17,19 @@
  */
 package pl.betoncraft.betonquest.config;
 
-import pl.betoncraft.betonquest.config.ConfigAccessor.AccessorType;
-import pl.betoncraft.betonquest.utils.Debug;
-import pl.betoncraft.betonquest.utils.Utils;
-
 import java.io.File;
 import java.util.HashMap;
 import java.util.Locale;
 import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import pl.betoncraft.betonquest.GlobalVariableID;
```

Comment:
Can you configure your IDE to order imports the same as intellij otherwise it'll be a constant juggling. Here is a view of what I have at the moment:
![image](https://user-images.githubusercontent.com/1607029/65007457-d49d8600-d938-11e9-8b0a-255068e1362f.png)

Reply:
I did the same in Eclipse, but i am not able to configure blank lines